CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing to jade
+
```

Comment:
We should include a section on asking for help in this guide.  I'd like to make it clear that it's OK to open an issue for **any** communication related to jade and that issues are not just for bugs.  It would also be good to have something to encourage people to a) search the docs first and b) post the entire error message, log and source code for reproducing any bugs.